extensions/podman/packages/extension/scripts/podman-download.ts

```diff
@@ -469,10 +276,10 @@ export class Podman5DownloadMachineOS {
 
     // now get the zstd entry from the arch manifest
     const amd64ZstdManifest = await this.getManifest(
-      `https://quay.io/v2/podman/machine-os/manifests/${amd64Manifest.digest}`,
+      `https://quay.io/v2/podman/machine-os${this.#wsl ? '-wsl' : ''}/manifests/${amd64Manifest.digest}`,
```

Comment:
here, probably inside the constructor, defined the github project name

then here we would just reference the repositoryLink (it avoids to forget one place) or if the repository is changing, etc